objgraph.py

```diff
@@ -143,6 +143,8 @@ def typestats(objects=None, shortnames=True, filter=None):
     Note that classes with the same name but defined in different modules
     will be lumped together if ``shortnames`` is True.
 
+    Use ``filter`` (a predicate) to remove undesired objects.
+
```

Comment:
Very good, only I'd also like to clarify whether the objects for which `filter()` returns `True` are included or excluded.

Perhaps something like

    If ``filter` is specified, it should be a function taking one argument and returning a boolean.
    Objects for which ``filter(obj)`` returns ``False`` will be ignored.

?